docs/make.jl

```diff
@@ -0,0 +1,28 @@
+using Documenter, Convex
+
+makedocs(;
+            modules = [Convex],
```

Comment:
These could all probably just be reduced to 4 spaces of absolute indentation rather than relative to a particular thing

Reply:
Ok, let me know if the new commit addresses this or if I misunderstood

Reply:
Yep, thanks!